packages/medusa/src/api/routes/admin/payment-collections/update-payment-collection.ts

```diff
@@ -0,0 +1,35 @@
+import { CreatePaymentCollectionInput } from "../../../../types/payment-collection"
+import { IsObject, IsOptional, IsString } from "class-validator"
+
+import { EntityManager } from "typeorm"
+import { PaymentCollectionService } from "../../../../services"
+
+export default async (req, res) => {
+  const { id } = req.params
+  const data = req.validatedBody as CreatePaymentCollectionInput
+
+  const paymentCollectionService: PaymentCollectionService = req.scope.resolve(
+    "paymentCollectionService"
+  )
+
+  const manager: EntityManager = req.scope.resolve("manager")
+  const payment_collection = await manager.transaction(
```

Comment:
nit(non-blocking): Camel case to follow general convention
```suggestion
  const paymentCollection = await manager.transaction(
```